charts/argo-cd/templates/argocd-applicationset/service.yaml

```
@@ -15,7 +15,7 @@ spec:
   ports:
   - name: {{ .Values.applicationSet.service.portName }}
     port: {{ .Values.applicationSet.service.port }}
-    targetPort: {{ .Values.applicationSet.containerPort }}
+    targetPort: 7000
```

```suggestion
    targetPort: webhook
```
Let's use the named port - then its less hardcoding ;)